### ISSUE TITLE
PSTV Internet Connection ON/OFF

### ISSUE THREAD

**Issue by LakiBR** [Ibrahim778/QuickMenuReborn#8]
Is there a possibility to make a plugin to add the wireless switch ON/OFF to quick menu?

**Comment by Ibrahim778** [comment]
Yes, that would certainly be possible

**Comment by LakiBR** [comment]
Do you think it would be too difficult to learn for someone who doesn't understand about plugin development? I would really like to activate this function on my PSTV, because every time I want to deactivate the internet, I have to go to the Settings app, and for that, I have to close the applications that are running. It would benefit many people who also want this function, as it is present in PSVita. I already found that config that changes in registry it's the net_flag (0=off; 1=on) without a need to reboot.

**Comment by Ibrahim778** [comment]
Do you have any prior experience developing with the psvita or any dev experience at all?

**Comment by LakiBR** [comment]
Unfortunately I don't have experience with dev yet. I have experience only in registry editing, game translation, filesystem in general and HeX editing. But programming, nothing at all. :/

**Comment by Ibrahim778** [comment]
Welp, you have a long way to go before trying something like this.
It's never too late to start though.

**Comment by LakiBR** [comment]
I  tried to do this, but without sucess. Time to give up. Thank you so much for your consideration and time.

**Comment by Ibrahim778** [comment]
What did you get stuck on?

**Comment by LakiBR** [comment]
I tried to compile all the vita SDK tools, but found errors. Searching on the web, found nothing.

Edit: Found the solution. Compiled all the tools.  I will try to use another plugin as base (I've the main code too), but IDK where to start with all these resources.

**Comment by Ibrahim778** [comment]
Very nice. It's great how you're trying
It'd be alot easier for us to talk and me to explain if you contact me somewhere else.
Like discord, you can find me at M Ibrahim#0197

**Comment by LakiBR** [comment]
Thanks for being so available, I added you on discord.

I tried to create this plugin so far, but really, I don't have enough technical knowledge for that. So unfortunately, I will have to give up on this project. I believe it's a simple thing to do (the scePowerSetUsingWireless function should work), but I still lack programming knowledge, which I'm far from developing.

Thank you for your support, and for the wonderful work. I hope that someday someone capable of doing so, will be able to compile this plugin.
